### PR TITLE
Lps 162464

### PR DESCRIPTION
**Pull request by Vy-Bui17**
LPS-162842 | LPS-162464

### CODE REVIEW

File: modules/apps/text-localizer/text-localizer-taglib/src/main/java/com/liferay/text/localizer/taglib/internal/address/util/AddressUtil.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
 package com.liferay.text.localizer.taglib.internal.address.util;
 
 import com.liferay.portal.kernel.model.Address;
+import com.liferay.portal.kernel.model.Country;
 import com.liferay.portal.kernel.model.Region;
 import com.liferay.portal.kernel.service.ServiceContextThreadLocal;
```
Comment (ces-sontruong) [inline modules/apps/text-localizer/text-localizer-taglib/src/main/java/com/liferay/text/localizer/taglib/internal/address/util/AddressUtil.java]: Please remove unused import statement

Comment (Vy-Bui17) [reply modules/apps/text-localizer/text-localizer-taglib/src/main/java/com/liferay/text/localizer/taglib/internal/address/util/AddressUtil.java]: updated.

**Comment by ces-sontruong** [comment]
Forwarded to liferay-usm#733. Thanks!

File: portal-impl/src/com/liferay/portal/model/impl/CountryImpl.java [hunk 2]
```diff
@@ -48,7 +48,17 @@ public String getNameCurrentLanguageId() {
 	@JSON
 	@Override
 	public String getNameCurrentValue() {
-		return getName(getLocale(_nameCurrentLanguageId));
+
```
Comment (ces-sontruong) [inline portal-impl/src/com/liferay/portal/model/impl/CountryImpl.java]: Please run formatSource

Comment (Vy-Bui17) [reply portal-impl/src/com/liferay/portal/model/impl/CountryImpl.java]: @sontruongces I've already run format source, what is the problem here?

Comment (ces-sontruong) [reply portal-impl/src/com/liferay/portal/model/impl/CountryImpl.java]: ci:test:sf failed in the PR: https://github.com/liferay-usm/liferay-portal/pull/733. Shouldn't have an empty line here (line 51).

Comment (Vy-Bui17) [reply portal-impl/src/com/liferay/portal/model/impl/CountryImpl.java]: ok, thanks for letting me know. Updated.

**Pull request closed without merging**